lib/proxy/log_buffer/decorator.rb

```diff
@@ -10,17 +10,15 @@ def initialize(logger, buffer = Proxy::LogBuffer::Buffer.instance)
 
     def add(severity, message = nil, progname = nil, backtrace = nil, a_module = 'core', &block)
```

Comment:
Ah I saw this logger is based upon Ruby's logger, I didn't know that - https://github.com/ruby/ruby/pull/1240